README.rst

```diff
@@ -340,9 +336,10 @@ like 'radosgw-agent'. This inclusion would be defined at the project level, like
     repos = {
         'ceph': {
             'extras': ['ceph-deploy', 'radosgw-agent'],
-            'deb': {
```

Comment:
I just confirmed that it is not possible to deal with combined repos for anything that is not a DEB. That would make this key redundant.
An alternative would be something like:

    'combined': ['wheezy', 'precise', 'jessie']